assets/src/components/amp-support/index.js

```diff
@@ -94,6 +95,9 @@ export function AMPSupport( props ) {
 						),
 					}
 				} />
+
+				{ <Notices data={ data } args={ args } ampValidatedPostCount={ ampValidatedPostCount } /> }
```

Comment:
There's no need to wrap the component in curly braces:

```suggestion
				<Notices data={ data } args={ args } ampValidatedPostCount={ ampValidatedPostCount } />
```